docs/feature_rgblight.md

```diff
@@ -30,6 +30,21 @@ At minimum you must define the data pin your LED strip is connected to, and the
 
 Then you should be able to use the keycodes below to change the RGB lighting to your liking.
 
+### Support on STM32
+
+STM32 chips support driving WS2812B LEDs using their SPI bus. You may want to do this to support both RGB Matrix and Underglow independently, at the same time. Driving via SPI also frees up 2 timers that are used in the alternative implemenetaion.
+
+If you want to use this feature, you have to turn it on specifically, as well as use specific pins and provide some extra information. SPI driven WS2812B LEDs require use of an SPI MOSI pin on your STM32 chip. 
+
+In addition to the values above, you must also define:
+
+|Define                |Description                                                    |
+|----------------------|---------------------------------------------------------------|
+|`RGBLIGHT_STM32_SPI`  |Enables the SPI DMA based RGB underglow feature                |
+|`WS2812_SPI`          |The SPI driver in use for your SPI MOSI pin (for example SPID2)|
```

Comment:
Could you explain also the RGB_DI_PIN variable?
I presume those 3 defines are the only things needed to configure the driver?

Reply:
We also need number of LEDs. I thought it was covered in the above part, but I can add it here too.

Reply:
Updated docs to address this

Reply:
Sorry for being too picky on this, but could this be moved under the configuration sections?
Also could you update this to reflect that there is drivers for stm32 now using one of the spi channels : https://docs.qmk.fm/#/hardware_drivers?id=ws2812-avr-only

Reply:
I think it actually belongs under usage. If you read the whole doc, the usage section explains how to turn it on for AVR so it should do the same for ARM. If we want to move it to configuration, we should move teh AVR setup as well I think.

Didn't realize that the driver doc was a thing - I'll update that as well

Reply:
Yes I see your point! 
I think its best to rename the Usage heading to Configuration and merge the current Configuration section there. The current Usage sections explains how to enable the feature and set the pin which is more like configuration. Usage for me is more what follows, ie color selection effects etc

Reply:
Addressed these issues and made revisions to the documentation